server/routes/users.js

```diff
@@ -3,167 +3,473 @@ var express = require('express');
 var router = express.Router();
 
 // Require database adapter file (not node-postgres directly)
-const db = require('../db')
+var knex = require('../db/knex');
+var util = require('../util');
+var Promise = require('bluebird');
```

Comment:
Hmmm we shouldn't need to import any packages? Is this necessary?

Reply:
fixed